packages/modules/cra-template-itwin-viewer/template/src/App.tsx

```diff
@@ -73,7 +73,7 @@ const App: React.FC = () => {
       ) : (
         isAuthorized && (
           <Viewer
-            projectId={process.env.REACT_APP_TEST_PROJECT_ID ?? ""}
+            projectId={process.env.REACT_APP_TEST_CONTEXT_ID ?? ""}
```

Comment:
should you change the prop name too?

Reply:
yes. Just another breaking change :-(. I'll update

Reply:
Sorry.. We need to figure out a better way to do these minor changes and not have to bump the major